layouts/css/base.styl

```diff
@@ -3,9 +3,11 @@
 // Base styles
 body
     box-sizing border-box
-    font 300 20px/1.5 'Source Sans Pro', Arial, sans-serif
-    color #000
+    font 400 15px/1.5 "source-sans-pro", sans-serif
+    color #333333
     margin 0
+    -webkit-font-smoothing antialiased
+    text-rendering optimizeLegibility
```

Comment:
Hmmmm, I think some base styles were lost from io.js here.

I suggest:

```styl
html
  -webkit-font-smoothing antialiased
  -moz-osx-font-smoothing grayscale
  -webkit-font-variant-ligatures none
          font-variant-ligatures none
```

Please remove `text-rendering optimizeLegibility` for text that isn't fairly large. It is very slow and can negatively impact page load times.